jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/actions/CaseActionsPresenter.java

```diff
@@ -64,26 +70,78 @@ protected void clearCaseInstance() {
     protected void loadCaseInstance(final CaseInstanceSummary cis) {
         view.updateListHeaders();
         newActionView.setCaseStagesList(cis.getStages());
+        caseDefinitions.clear();
+        caseService.call(
+                (List<CaseDefinitionSummary> definitions) -> {
+                    final List<String> definitionNames = new ArrayList<>();
+                    for (CaseDefinitionSummary summary : definitions) {
+                        definitionNames.add(summary.getName());
+                        caseDefinitions.put(summary.getName(), summary);
+                    }
+                    Collections.sort(definitionNames);
+                    newActionView.setCaseDefinitions(definitionNames);
+                }
+        ).getCaseDefinitions();
```

Comment:
I think this call is incorrect, we should get all the processes that are not cases, which is returned by _CaseServicesClient.findProcessesByContainerId_.